rest/src/main/groovy/whelk/rest/api/Crud.groovy

```diff
@@ -207,6 +208,7 @@ class Crud extends HttpServlet {
 
     static void displayInfo(HttpServletResponse response) {
         def info = [:]
+        // FIXME: de-KBV/Libris-ify?
```

Comment:
Nice to have.
Should at some point be replaced with something that is actual linked data. 
`:DataCatalog` or something

Reply:
Just take from `site_config.json`?